runc/plan.sh

```diff
@@ -1,12 +1,44 @@
 pkg_name="runc"
 pkg_origin="core"
-pkg_version=0.1.1
+pkg_version=1.0.0-rc10
```

Comment:
I'm assuming we are OK packaging an `-rc10` release in this case to make sure that people don't regress in functionality since we were technically previously shipping whatever was on master when the last package was built?

Reply:
For what it's worth, Ubuntu provides it in their packages: https://packages.ubuntu.com/bionic/runc

Reply:
:shipit: